pages/linux/rpcinfo.md

```diff
@@ -0,0 +1,23 @@
+# rpcinfo
+
+> Makes an RPC call to an RPC server and reports what it finds.
+
+- Show full table of all RPC services registered on localhost:
+
+`rpcinfo`
+
+- Show concise table of all RPC services registered on localhost:
+
+`rpcinfo -s {{localhost}}`
+
+- Display table of statistics of rpcbind operations on localhost:
+
+`rpcinfo -m`
+
+- Display list of entries of given service name (mountd) and version number (2) on localhost:
+
+`rpcinfo -l {{localhost}} {{mountd}} {{2}}`
```

Comment:
Maybe we should change this example from localhost to some other IP, so that we have different examples.